WordPress/src/main/java/org/wordpress/android/ui/prefs/AppPrefs.java

```diff
@@ -1683,4 +1687,19 @@ public static Boolean getShouldHideBlazeOverlay() {
     public static void setShouldHideBlazeOverlay(final boolean isHidden) {
         setBoolean(DeletablePrefKey.SHOULD_HIDE_BLAZE_OVERLAY, isHidden);
     }
+
+    public static Boolean getShouldShowJetpackSocialNoConnections(final int siteId, final JetpackSocialFlow flow) {
+        return prefs().getBoolean(mapShouldShowJetpackSocialNoConnectionsKey(siteId, flow), false);
```

Comment:
⚠️ The default here should be `true` since we use this pref to actually hide the "No Connections" UI, so if it's absent it would mean that it is not hidden, so it _should show it_.

Reply:
Good catch, thanks!